frontend/src/app/main/ui/workspace/shapes/text/new_editor.cljs

```diff
@@ -0,0 +1,259 @@
+;; This Source Code Form is subject to the terms of the Mozilla Public
+;; License, v. 2.0. If a copy of the MPL was not distributed with this
+;; file, You can obtain one at http://mozilla.org/MPL/2.0/.
+;;
+;; Copyright (c) KALEIDOS INC
+
+(ns app.main.ui.workspace.shapes.text.new-editor
+  (:require-macros [app.main.style :as stl])
+  (:require
+   ["./new_editor_impl.js" :as impl]
```

Comment:
I think this should follow the same naming as used in functions: `v2_editor_impl.js`

Reply:
The same applies to the namespace `new-editor`